_includes/binary2.html

```diff
@@ -23,7 +23,7 @@
         <div class="opcol">AND</div>
         <div class="opcol">AND</div>
         <div class="opcol">AND</div>
-    </div><input type="button" value="Calculate" onClick="set_bits();" class="ok" style="margin-top: 22px"><br style="clear: left">
+   
```

Comment:
@CodeSarthak Please remove this blank space. It's good to approve after these small change.